src/jit/lower.h

```diff
@@ -189,6 +190,7 @@ class Lowering : public Phase
     void TreeNodeInfoInitReturn(GenTree* tree);
     void TreeNodeInfoInitShiftRotate(GenTree* tree);
     void TreeNodeInfoInitCall(GenTreeCall* call);
+    void TreeNodeInfoInitCmp(GenTreePtr tree);
     void TreeNodeInfoInitStructArg(GenTreePtr structArg);
```

Comment:
Note that one of the reasons I didn't rename this before is because this function does a combination of setting TreeNodeInfo data as well as doing some morphing optimizations which seem more like "lowering", whereas the rest of the "TreeNodeInfoInit*" functions are purely (I think) setting the TreeNodeInfo. I don't know if it's possible to split these two functionalities to make it better factored/named.